templates/confirm.html

```diff
@@ -3,57 +3,80 @@
 {% block title %}確認{% endblock %}
 
 {%- block content %}
-  <h1>確認のページです</h1>
-  {{salt}}
-  {{ketchup}}
-  {{cheese}}
-  {{butter}}
-  <div class="container mt-5">
-    <div class="h5">これはスプレッドシートに書き込まれるか確認するためのものです</div>
-    <form action="/result" method="post">
-      <div class="row position-absolute top-50 start-50 translate-middle">
-        <div class="col-6 mb-5">
-          <label for="validationCustom04" class="form-label">チーズ</label>
-          <select class="form-select" aria-label="Default select example" name="cheese">
-            <option selected="selected">0</option>
-            <option value="1">1</option>
-            <option value="2">2</option>
-            <option value="3">3</option>
-          </select>
-        </div>
-        <div class="col-6 mb-5">
-          <label for="validationCustom04" class="form-label">バター醤油</label>
-          <select class="form-select" aria-label="Default select example" name="butter">
-            <option selected="selected">0</option>
-            <option value="1">1</option>
-            <option value="2">2</option>
-            <option value="3">3</option>
-          </select>
-        </div>
-        <div class="col-6">
-          <label for="validationCustom04" class="form-label">塩</label>
-          <select class="form-select" aria-label="Default select example" name="salt">
-            <option selected="selected">0</option>
-            <option value="1">1</option>
-            <option value="2">2</option>
-            <option value="3">3</option>
-          </select>
-        </div>
-        <div class="col-6">
-          <label for="validationCustom04" class="form-label">ケチャップ</label>
-          <select class="form-select" aria-label="Default select example" name="ketchup">
-            <option selected="selected">0</option>
-            <option value="1">1</option>
-            <option value="2">2</option>
-            <option value="3">3</option>
-          </select>
-        </div>
-        <button type="submit" class="btn btn-primary mt-5">注文する</button>
-      </div>
-    </form>
-  </div>
-{%- endblock %}
+  <h1 class="text-center">確認のページです</h1>
+  <div class="container">
+    <div class="container">
+      <div class="row">
+        <div class="me-auto">
+          <p class="text-center h5 mt-3">ご注文いただきます商品は、こちらでお間違いないでしょうか？</p>
+          <div class="container-fluid">
+            <div class="row mt-4 justify-content-center">
+              <div class="center-block">
+                <form action="/result" method="post">
+                  <div class="mx-auto" style="width:200px;">
+                    <table class="table">
+                      <tbody>
+                        {%if salt != 0%}
```

Comment:
下のようにスペースを入れると整形がうまくいきました。
他の部分も修正お願いします。

```suggestion
                        {% if salt != 0 %}
```

Reply:
了解です！